projects/laji/src/app/+taxonomy/taxon/info-card/info-card.component.ts

```diff
@@ -148,8 +148,12 @@ export class InfoCardComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   updateRoute(id: string, tab = this.activeTab, context = this.context, replaceUrl = false) {
-    this.selectedTab = tab;
+    if (this.activeTab !== 'endangerment' && this.activeTab !== 'invasive') {
+      this.loadedTabs.reset();
+    } 
```

Comment:
changing tab causes updateRoute so resetting loadedTabs on almost all tabs is not ideal. (idea with loaded tabs is to keep data for one specific taxon always in memory and not to re-fetch the same data on every tab change)